packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -0,0 +1,100 @@
+import json
+import requests
+
+RANGE_ENDPOINT = '/system/v1/logs/v1/range/'
+STREAM_ENDPOINT = '/system/v1/logs/v1/stream/'
+
+
+def validate_json_entry(entry):
+    required_top_level_fields = ['fields', 'cursor', 'monotonic_timestamp', 'realtime_timestamp']
+
+    assert isinstance(entry, dict), 'Entry must be dict'
+    assert entry, 'Entry cannot be empty'
+
+    for required_field in required_top_level_fields:
+        assert required_field in entry, 'Field {} must be in entry. Got {}'.format(required_field, entry)
+
+    assert entry['fields'], '`fields` cannot be empty dict. Got {}'.format(entry)
+
+
+def validate_sse_entry(entry, stream=False):
+    lines = entry.split('\n')
+    assert len(lines) >= 2, 'Expect an least 2 lines of entry. Got {}'.format(entry)
+
+    data_line = {
+        True: lines[1],
+        False: lines[0]
+    }.get(stream)
+
+    entry_json = json.loads(data_line.lstrip('data: '))
+    validate_json_entry(entry_json)
+
+    if stream:
+        cursor = lines[0].lstrip('id: ')
+        assert cursor == entry_json['cursor'], 'In streaming mode an entry must have id: Got {}'.format(entry)
+
+
+def test_log_text(cluster):
+    for node in cluster.masters + cluster.all_slaves:
+        response = cluster.get(path=RANGE_ENDPOINT + '?limit=10', node=node)
+        assert response.ok
```

Comment:
should do something like `assert response.ok, "Response code {}, body: {}".format(response.status_code, response.body)`. Currently if that fails, we'll see that the response status code was not equal  to 200, but won't see the response which means it'll be really hard to debug (esp. if it is a flaky failure).